source/rust_verify/src/main.rs

```diff
@@ -110,6 +110,27 @@ pub fn main() {
 
     let pervasive_path = our_args.pervasive_path.clone();
 
+    if our_args.error_report {
+        let mut args: Vec<String> = std::env::args().collect();
+        args.remove(0);
+
+        let index = args.iter().position(|x| *x == "--error-report").unwrap();
+        args.remove(index);
+
+        let verus_path = std::env::current_exe().unwrap();
+        let exe = verus_path.parent().unwrap().join("../../error_report/target/debug/error_report");
+
+        let mut res = std::process::Command::new(exe)
+            // .current_dir(std::env::current_dir().unwrap())
```

Comment:
Out of curiosity, why is this comment here? I'd assume `current_dir` to be set to the current dir anyways (which is what this comment implies)